src/components/main/MainMenu.tsx

```diff
@@ -47,21 +56,72 @@ const MainMenu = (): JSX.Element => {
   const navigate = useNavigate();
   const { pathname } = useLocation();
   const { data: member } = useCurrentUserContext();
+  const { mutate: postBug } = mutations.usePostBug();
 
+  const [isSentryFormOpen, setIsSentryFormOpen] = useState(false);
   const theme = useTheme();
   const iconColor = theme.palette.action.active;
 
   const goTo = (path: string) => {
     navigate(path);
   };
 
+  useEffect(() => {
+    if (isSentryFormOpen) {
+      setTimeout(() => {
+        const ele: HTMLButtonElement | null = document.querySelector(
+          '.form-submit > .btn',
+        );
+        const nameInput: HTMLInputElement | null =
+          document.querySelector('#id_name');
+        const emailInput: HTMLInputElement | null =
+          document.querySelector('#id_email');
+        const commentsInput: HTMLInputElement | null =
+          document.querySelector('#id_comments');
+
+        if (ele) {
+          const originalClick: any = ele.onclick;
+          ele.onclick = null;
+          ele?.addEventListener('click', (e) => {
+            e?.preventDefault();
+            originalClick?.(e);
+            postBug({
+              name: nameInput?.value || '',
+              email: emailInput?.value || '',
+              details: commentsInput?.value || '',
+            });
+          });
+        }
+      }, 1000);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isSentryFormOpen]);
```

Comment:
I think it would work if you create a new component `ReportABug.tsx` that would contain this `useEffect` as well as the `openBugReport` right? So we don't overflow `MainMenu.tsx`.